azure-kusto-data/setup.py

```diff
@@ -13,7 +13,7 @@
 # a-b-c => a.b.c
 NAMESPACE_NAME = PACKAGE_NAME.replace("-", ".")
 
-with open(path.join(PACKAGE_FOLDER_PATH, "_version.py"), "r") as fd:
+with open(path.join(PACKAGE_FOLDER_PATH, "_version.py")) as fd:
```

Comment:
why have you removed the "r"

Reply:
I used pyupgrade for these changes, and they recommend removing "r" as the mode since it is now the default.

I thought that this is pretty cool and left it that way, but I could revert if you prefer